<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="sZoom">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>370</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDial" name="dVitesse">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lSpeed">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bResetZoom">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetZoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="lZoomValue">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bCursor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Cursor</string>
    </property>
   </widget>
   <widget class="QLabel" name="lZoomValuesScale">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scale: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lCursorState">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>16</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>16</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="glChart">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QSlider" name="sPosition">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>390</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lPosition">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPositionValue">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bSignalTest">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Test</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
